druid/examples/identity.rs

```diff
@@ -53,6 +53,26 @@ fn color_eq(one: &Color, two: &Color) -> bool {
     one.as_rgba_u32() == two.as_rgba_u32()
 }
 
+fn split_rgba(rgba: &Color) -> (f64, f64, f64, f64) {
+    let rgba = rgba.as_rgba_u32();
+    (
+        (rgba >> 24) as f64 / 255.0,
+        ((rgba >> 16) & 255) as f64 / 255.0,
+        ((rgba >> 8) & 255) as f64 / 255.0,
+        (rgba & 255) as f64 / 255.0,
+    )
+}
+
+fn color_average(one: &Color, two: &Color) -> Color {
```

Comment:
This is a low pass filter rather than a average, I would suggest naming this appropriately